// async function divideAsync(a, b) {
//     if (b === 0) {
//         throw new Error("Division by zero is not allowed");
//     }
//     return a / b;
// }

// async function performDivision() {
//     try {
//         let result = await divideAsync(10, 2);
//         console.log("Result of division:", result);
//     } catch (error) {
//         console.error("Error:", error);
//     }
// }

// performDivision();
